apps/main/src/dao/components/PageUser/UserGaugeVotesTable/index.tsx

```diff
@@ -59,7 +59,7 @@ const UserGaugeVotesTable = ({ userAddress, tableMinWidth }: UserGaugeVotesTable
         <TableRowWrapper key={index} columns={GAUGE_VOTES_LABELS.length} gridTemplateColumns={gridTemplateColumns}>
           <TableData className="align-left">{gaugeVote.gaugeName}</TableData>
           <TableData className={`right-padding ${userGaugeVotesSortBy.key === 'timestamp' ? 'sortby-active' : ''}`}>
-            {formatDateFromTimestamp(convertToLocaleTimestamp(gaugeVote.timestamp))}
```

Comment:
The original code didn't divide by 1000, but the new function will. Is this still correct?

Reply:
given:
https://github.com/curvefi/curve-frontend/blob/788a95fcad2d0e0a5a7480c204f70becfb631323/packages/ui/src/utils/utilsDate.ts#L98-L99

I did not change anything functionally right? The call is literally the same, `formatDateFromTimestamp` and `convertToLocaleTimestamp` aren't changed